### ISSUE TITLE
BlockCompressedPowered -> ???

### ISSUE THREAD

**Issue by bs2609** [ModCoderPack/MCPBot-Issues#537]
This is the redstone block class. I can kinda see the desire for a more generic name, given the only custom logic is to make it emit a redstone signal, but the name isn't really very helpful.

I think it should have "redstone" in the name, at least. Possibly just `BlockRedstone` would be fine?

Any other good ideas?

**Comment by williewillus** [comment]
If it's not used for anything else then yeah BlockRedstone makes perfect sense

**Comment by Pokechu22** [comment]
Let me make a (somewhat weak) argument that it should remain the same:

In the [april fools release 1.RV-Pre1](https://minecraft.gamepedia.com/1.RV-Pre1), there is a block called the "USB Charger Block".  Now, before I discuss this block, let me link to the burger data for this version: [full HTML](http://pokechu22.github.io/Burger/1.RV-Pre1.html), [full JSON](http://pokechu22.github.io/Burger/1.RV-Pre1.json), [1.9.1 diff HTML](http://pokechu22.github.io/Burger/diff_1.9.1_and_1.RV-Pre1.html), and [1.9.1 diff JSON](http://pokechu22.github.io/Burger/diff_1.9.1_and_1.RV-Pre1.json).

But the only important one is the full JSON.  Let me make two quotes, both from the block section:

```json
                "redstone_block": {
                    "class": "anm", 
                    "display_name": "Block of Redstone", 
                    "field": "cn", 
                    "hardness": 5, 
                    "name": "blockRedstone", 
                    "numeric_id": 152, 
                    "text_id": "redstone_block"
                }, 
```
```json
                "usb_charger": {
                    "class": "anm", 
                    "display_name": "USB Charger Block", 
                    "field": "df", 
                    "hardness": 3, 
                    "name": "usbCharger", 
                    "numeric_id": 250, 
                    "text_id": "usb_charger"
                }, 
```

Basically, yes, it _was_ used for something else, albeit in a joke snapshot.  It's not used for something else currently.  But I think leaving it as the current name is _justifiable_.

I also don't particularly like `BlockRedstone` - that sounds like the wire too much to me.  The actual text ID is `minecraft:redstone_block`; if `BlockRedstoneBlock` didn't sound so janky I'd recommend it but as such I'm not sure what it should be changed to, if it should be changed.

**Comment by diesieben07** [comment]
I think @Pokechu22 just wanted to brag about knowing that :D

**Comment by bs2609** [comment]
A generic name _could_ be okay, if that name wasn't `BlockCompressedPowered` (which is not something I've ever called a USB charger).

**Comment by Pokechu22** [comment]
How about `BlockSimplePowered` or `BlockSimplePowerSource`?  (The `compressed` portion definitely does not apply, as this is completely unrelated to `BlockCompressed`)

**Comment by kashike** [comment]
- [x] `net/minecraft/block/BlockCompressedPowered` → `net/minecraft/block/BlockRedstone`
